### PR TITLE
#3 Registration form

### PR DESCRIPTION
**Pull request by Oleg-pnu**
https://todoist.com/app/project/2309837149/task/6736508242

### CODE REVIEW

File: src/Components/Navibar.js [hunk 1]
```diff
@@ -13,7 +14,13 @@ a, .navbar - brand, .navbar - nav, .nav - link {
...
 
-function NaviBar() {
+export default function NaviBar() {
+
+	const [show, setShow] = useState(false);
+
+	const handleClose = () => setShow(false);
+	const handleShow = () => setShow(true);
+
 	return (
```
Comment (kostiantynShakhzadov) [inline src/Components/Navibar.js:24]: @Oleg-pnu please change the tab size rendering preference for the current project on GitHub, as it is hard to review the code -> https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference

File: src/Components/Navibar.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { hover } from "@testing-library/user-event/dist/hover";
-import React from "react";
-import { Navbar, Nav, Button, Container } from 'react-bootstrap';
+import React, { useState } from "react";
+import { Navbar, Nav, Button, Container, Form } from 'react-bootstrap';
```
Comment (kostiantynShakhzadov) [inline src/Components/Navibar.js]: @Oleg-pnu please stick to one style of quotes (single or double) throughout the project.

File: src/Components/Navibar.js [hunk 3]
```diff
@@ -28,15 +35,34 @@ function NaviBar() {
...
+							<Form.Label>Email Adress</Form.Label>
+							<Form.Control type="email" placeholder="Enter email" />
+							<Form.Text className="text-muted">We'll never share your email whith anyone else.</Form.Text>
+						</Form.Group>
+						<Form.Group controlId="formBasicPassword">
+							<Form.Label>Password</Form.Label>
+							<Form.Control type="password" placeholder="Enter password" />
+						</Form.Group>
+						<Form.Group controlId="formBasicCheckbox">
+							<Form.Check type="checkbox" label="Remeber me" />
```
Comment (kostiantynShakhzadov) [inline src/Components/Navibar.js]: @Oleg-pnu typo in the label name, one `m` is missing

File: src/Components/Navibar.js [hunk 4]
```diff
@@ -28,15 +35,34 @@ function NaviBar() {
...
+			<Modal show={show} onHide={handleClose}>
+				<Modal.Header closeButton>
+					<Modal.Title>Log in</Modal.Title>
+				</Modal.Header>
+				<Modal.Body>
+					<Form>
+						<Form.Group controlId="formBasicEmail">
+							<Form.Label>Email Adress</Form.Label>
+							<Form.Control type="email" placeholder="Enter email" />
+							<Form.Text className="text-muted">We'll never share your email whith anyone else.</Form.Text>
```
Comment (kostiantynShakhzadov) [inline src/Components/Navibar.js]: @Oleg-pnu typo in `with` word.

File: src/Components/Navibar.js [hunk 5]
```diff
@@ -28,15 +35,34 @@ function NaviBar() {
...
 				</Navbar>
 			</Styles>
+			<Modal show={show} onHide={handleClose}>
+				<Modal.Header closeButton>
+					<Modal.Title>Log in</Modal.Title>
+				</Modal.Header>
+				<Modal.Body>
+					<Form>
+						<Form.Group controlId="formBasicEmail">
+							<Form.Label>Email Adress</Form.Label>
```
Comment (kostiantynShakhzadov) [inline src/Components/Navibar.js]: @Oleg-pnu typo in `Address` word.

**Review by kostiantynShakhzadov — COMMENTED** [review]
Please check the comments and fix them before I'll merge them. Tag me, when completed.

File: src/Components/Navibar.js (hunk 5 above)
Comment (Oleg-pnu) [reply src/Components/Navibar.js]: Done.

File: src/Components/Navibar.js (hunk 1 above)
Comment (Oleg-pnu) [reply src/Components/Navibar.js:24]: I put four instead of eight.

**Review by kostiantynShakhzadov — APPROVED** [review]

**Pull request merged**